stdlib/public/Concurrency/Deque.swift

```diff
@@ -0,0 +1,502 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2020-2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Swift
+
+@available(SwiftStdlib 5.5, *)
+@frozen
+@usableFromInline
+struct _Deque<Element> {
```

Comment:
This is a lot of ABI surface area to take on. Is this the Deque implementation for the Swift Collections package, or something bespoke?

Reply:
this is 100% taken from the Swift Collections package, and nothing should be exposed for this to external callers